### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ Button _ Main header title not read

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3144]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Button
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _ Modify Modules _ Button _ Main header title not read

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Navigate to ‘Information’ button within main header region.

3. Issue Type: Manual
Observed Behavior:
On accessing the first interactive Ui element within the main header region across the application, e.g. ‘Information’ button, the respective main header title/text e.g. ‘Modules’ is not announced by the screen reader. 
Screenshot:
 
Case 2:
 

Case 3:
 

Case 4:
 
Case 6:
 

Expected Behavior:
On accessing the first interactive Ui element within the main header region across the application, e.g. ‘Information’ button, the respective main header title/text must be announced by the screen reader along with other necessary information. 

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1 (Level A) _ Modify Modules _ Button _ Main header title not read.docx](https://github.com/user-attachments/files/16659997/ACC-264.1.Level.A._.Modify.Modules._.Button._.Main.header.title.not.read.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659998/Issue.processing-09.2021.002.docx)
